src/app/ubs-admin/components/ubs-admin-order-client-info/ubs-admin-order-client-info.component.ts

```diff
@@ -0,0 +1,63 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { FormGroup, FormControl, Validators } from '@angular/forms';
+import { Subject } from 'rxjs';
+import { takeUntil } from 'rxjs/operators';
+import { IUserInfo } from '../../models/ubs-admin.interface';
+import { OrderService } from '../../services/order.service';
+import { LocalStorageService } from '@global-service/localstorage/local-storage.service';
+
+@Component({
+  selector: 'app-ubs-admin-order-client-info',
+  templateUrl: './ubs-admin-order-client-info.component.html',
+  styleUrls: ['./ubs-admin-order-client-info.component.scss']
+})
+export class UbsAdminOrderClientInfoComponent implements OnInit, OnDestroy {
+  userInfo: IUserInfo;
+  customerInfoForm: FormGroup;
+  customerName: FormControl;
+  totalUserViolations: number;
+  userViolationForCurrentOrder: number;
+  orderId = 893;
+  public currentLanguage: string;
```

Comment:
You should follow the single convention
At some places, you have that `public` access modifier, at some not
You should make it the same everywhere

Reply:
Done